docs/src/other/glossary.md

```diff
@@ -110,6 +110,12 @@ The Platform.sh CLI configures Drush aliases for you on your local environment
 (via `platform get` or `platform drush-aliases`).
 You can also configure them manually.
 
+## Environment
+An [environment](../environments/_index.md) is a standalone copy of your site,
+complete with code, data, and running services.
+Your [default branch](../environments/_index.md#default-environment) is the production environment,
+and you can set up any other branch as an identical testing environment.
```

Comment:
:hammer_and_wrench: It's not always identical, which is helpful for testing new features.
```suggestion
and you can set up any other branch as a testing environment.
```